### DISCUSSION TITLE
Possible typo in HiFiGan's max_steps value

### DISCUSSION THREAD

**Discussion by eyentei** [NVIDIA-NeMo/NeMo#4431] [Q&A] [answered]
Is there any particular reason why hifigan's max_steps value is 25,000,000 [here](https://github.com/NVIDIA/NeMo/blob/main/examples/tts/conf/hifigan/hifigan.yaml#L67)  as opposed to 2,500,000 [in the original repo](https://github.com/jik876/hifi-gan/issues/4#issuecomment-719941567)? I've personally had better results with 2,500,000 (model converged way faster). Is it a typo?

**Comment by ericharper** [comment]
@redoctopus could you answer this one?

**Comment by redoctopus** [comment, marked as answer]
Ah yeah, that looks like a typo. I'll make a note and put in a fix soon, thanks for the catch!

**Reply by redoctopus** [reply]
PR #4446 merged :)
